$NetBSD$

Move the configuration file up one directory, no point in having a
directory containing one file.

--- configure.orig	2011-06-12 10:58:24.000000000 +0000
+++ configure
@@ -5798,7 +5798,7 @@ fi
 
 # where to find the configuration file
 # TODO: this should probably be fixed to be nicer
-CONFIGFILE=`eval echo $sysconfdir/cvsd/cvsd.conf | \
+CONFIGFILE=`eval echo $sysconfdir/cvsd.conf | \
             sed "s%^NONE/%${prefix}/%" | \
             sed "s%^NONE/%${ac_default_prefix}/%"`
 
